src/components/Panels/PropertiesPanel/PropertiesPanel.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        id="moduleProperty"
+        class="moduleProperty noSelect effect1 properties-panel draggable-panel draggable-panel-css guide_2"
+    >
+        <PanelHeader :header-title="$t('simulator.panel_header.properties')" />
+        <div class="panel-body">
+            <div id="moduleProperty-inner">
+                <div id="moduleProperty-header">{{ panelBodyHeader }}</div>
+                <PanelType1 v-if="panelType == 1" />
+                <PanelType2 v-if="panelType == 2" />
+                <PanelType3
+                    v-if="panelType == 3"
+                    :key="toRaw(propertiesPanelObj)"
+                    :data="toRaw(propertiesPanelObj)"
+                />
+                <HelpButton
+                    :key="toRaw(propertiesPanelObj)"
+                    :data="toRaw(propertiesPanelObj)"
+                />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { ref, toRaw } from '@vue/reactivity'
+import simulationArea from '#/simulator/src/simulationArea'
+import PanelHeader from '../Shared/PanelHeader.vue'
+import HelpButton from '../Shared/HelpButton.vue'
+import { onMounted } from 'vue'
+import {
+    checkPropertiesUpdate,
+    hideProperties,
+    prevPropertyObjGet,
+    prevPropertyObjSet,
+} from '#/simulator/src/ux'
+import { layoutModeGet } from '#/simulator/src/layoutMode'
+import PanelType1 from './PanelTypes/PanelType1.vue'
+import PanelType2 from './PanelTypes/PanelType2.vue'
+import PanelType3 from './PanelTypes/PanelType3.vue'
+
+const panelBodyHeader = ref('PROJECT PROPERTIES')
+const propertiesPanelObj = ref(undefined)
+const panelType = ref(2) // default is panel type 2 (project properties)
+onMounted(() => {
+    // checks for which type of properties panel to show
+    setInterval(showPropertiesPanel, 100)
+})
+
+function showPropertiesPanel() {
+    checkPropertiesUpdate()
+    if (toRaw(propertiesPanelObj.value) == simulationArea.lastSelected) return
+    prevPropertyObjSet(simulationArea.lastSelected)
+    propertiesPanelObj.value = simulationArea.lastSelected
+    // there are 3 types of panel body for Properties Panel
+    // depending upon which is last selected
+    // 1. Properties Panel in Layout mode
+    // 2. Properties Panel showing Project Properties
+    // 3. Properties Panel showing Circiut Element Properties
+
+    if (layoutModeGet()) {
+        panelType.value = 1
```

Comment:
This would be the Layout Panel

<img width="169" alt="image" src="https://user-images.githubusercontent.com/61665451/178153200-716ebc27-3d63-460d-91e5-7670064aa34d.png">

and the properties of the layout elements on selecting

<img width="551" alt="image" src="https://user-images.githubusercontent.com/61665451/178153557-f30d4ad7-8507-45b8-9275-b3d3e435a38f.png">